scripts/invalid_site_rechecker.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+for f in $(awk -F " " '/url:/ {print $2}' _data/sites.yml)
```

Comment:
the path should be ../_data/sites.yml

Reply:
Yes, I mentioned above - I took the path assumption from check_alphabetical_sorting.sh which is in the same directory but also uses _data/sites.yml instead of ../_data/sites.yml. I wasn't sure how these scripts are used, if they are moved, etc. Thanks for the correction.